packages/mgt-spfx/config/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/spfx-build/config.2.0.schema.json",
+  "version": "2.0",
```

Comment:
For my clarity, is this version the same as the version of the package, as in, should it be updated each time the version changes?

Reply:
I think that would make the most sense to keep everything in sync.